src/components/CompleteProfile/CompleteProfile.tsx

```diff
@@ -0,0 +1,419 @@
+										type='file'
+										accept='image/png, image/jpeg'
+										onChange={handlePhotoUpload}
+									/>
+									{photoFile && (
+										<Button
+											type='button'
+											hasOutline
+											onClick={handleRemovePhoto}
+										>
+											Remove Photo
+										</Button>
+									)}
+								</div>
+							</div>
+						</div>
+					</details>
+
+					<details open>
+						<summary className='text-h6'>
+							<span>More Information:</span>
+						</summary>
+
+						<div className='details-content-wrapper'>
+							<div id='details-information-grid'>
+								<div>
+									<label htmlFor='linkedInProfile'>LinkedIn profile</label>
+									<input
+										id='linkedInProfile'
+										name='linkedInProfile'
+										type='url'
+										className='text-input'
+									/>
+								</div>
+								<div>
+									<label htmlFor='githubProfile'>GitHub profile</label>
+									<input
+										id='githubProfile'
+										name='githubProfile'
+										type='url'
+										className='text-input'
+									/>
+								</div>
+								<div>
+									<label htmlFor='portfolio'>Site/portfolio</label>
+									<input
+										id='portfolio'
+										name='portfolio'
+										type='url'
+										className='text-input'
+									/>
+								</div>
+								<div id='details-information-skills'>
+									<label htmlFor='skill'>
+										<span>Your skills</span>
+										<div id='skills'>
+											{skills.map((skill) => (
+												<span key={skill.id}>
+													{skill.name}
+													<button
+														type='button'
+														aria-label='Remove Skill'
+														onClick={() => handleRemoveSkill(skill.id)}
+													>
+														x
+													</button>
+												</span>
+											))}
+											<input
+												id='skill'
+												ref={skillInputRef}
+												onKeyDown={handleInputSkill}
+												placeholder={skills.length === 0
+													? 'Insert some of your skills and separate them with commas (e.g.: JavaScript, HTML, CSS)'
+													: 'type here ...'}
+											/>
+										</div>
+									</label>
+								</div>
+
+								<div id='details-social-accounts'>
+									<p>Other social accounts (optional):</p>
+									<div>
+										{socialIcons.map(
+											(socialIcon) =>
+												socialIcon.visible && (
+													<button
+														aria-label={`Add ${socialIcon.name} account`}
+														key={socialIcon.name}
+														type='button'
+														onClick={() => handleAddSocialAccount(socialIcon.name)}
+													>
+														{socialIcon.element}
+													</button>
+												)
+										)}
+									</div>
+								</div>
+								<div id='details-social-inputs'>
+									{socialAccountInputs.map((socialInput) => (
+										<div>
+											<label htmlFor={socialInput}>
+												{socialInput}
+												<button
+													type='button'
```

Comment:
```suggestion
													type='button'
																										aria-label={`Close ${socialInput} input`}
```

These button gets announced as only "button"

Reply:
I need to start using a screen reader for catching these.